charts/nri-kube-events/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Kube Events
 name: nri-kube-events
-version: 1.6.1
+version: 1.6.2
```

Comment:
should we change the minor? 
(Not that important since only the forwarder changed and not the main app)

Reply:
+1 for changing the minor, I prefer to bump minor in charts whenever the minor appVersion of any component is bumped as well (and same goes for patch or major).